odoo/service/server.py

```diff
@@ -1353,18 +1353,21 @@ def stop(self):
 server = None
 server_phoenix = False
 
+
 def load_server_wide_modules():
     from odoo.modules.module import load_openerp_module  # noqa: PLC0415
-    for m in config['server_wide_modules']:
-        try:
-            load_openerp_module(m)
-        except Exception:
-            msg = ''
-            if m == 'web':
-                msg = """
-The `web` module is provided by the addons found in the `openerp-web` project.
-Maybe you forgot to add those addons in your addons_path configuration."""
-            _logger.exception('Failed to load server-wide module `%s`.%s', m, msg)
+    with gc.disable_gc():
```

Comment:
Context managers tend to use present continuous instead of present simple tense.  That makes it more intuitive that we are disabling the gc while being in the context
```suggestion
    with gc.disabling_gc():
```